Pipfile

```diff
@@ -6,15 +6,15 @@ verify_ssl = true
 [dev-packages]
 pytest = "*"
 pylint = "*"
-molecule = "*"
+molecule = "24.9.0"
```

Comment:
@dkinzer Does molecule need to be pinned?

Reply:
@sensei100 you're right it shouldn't be.